src/hardware/BLEMIDI_ESP32_NimBLE.h

```diff
@@ -128,17 +128,17 @@ bool BLEMIDI_ESP32_NimBLE::begin(const char* deviceName, BLEMIDI_Transport<class
                                                      );
 
     _characteristic->setCallbacks(new MyCharacteristicCallbacks(this));
+
+    auto _security = new NimBLESecurity();
+    _security->setAuthenticationMode(ESP_LE_AUTH_BOND);
```

Comment:
Is this needed? When I comment this out, pairing still works in Windows.

Reply:
If that part is not there, pairing will be successful, but once disconnected, the next automatic reconnection will fail and you will have to delete and repair the device each time.